docs/tutorial/part-one/index.md

```diff
@@ -30,7 +30,7 @@ You should see something like:
 
 ![Check if node.js/npm is installed](check-versions.png)
 
-Gatsby supports versions of Node back to v6 and npm to v3.
+Gatsby supports versions of Node back to v7.6 and npm to v3.
```

Comment:
I just tried v7.x
When I open localhost:9000 I get the following error:
```
TypeError: promisify is not a function

  - index.js:21 getHandlers
    [tutorial-part-one]/[serve-handler]/src/index.js:21:8

  - index.js:499 module.exports
    [tutorial-part-one]/[serve-handler]/src/index.js:499:19

  - serve.js:16 Server.http.createServer
    [tutorial-part-one]/[gatsby]/dist/commands/serve.js:16:57
```
`promisify` is a new utility function from Node 8.
So the minimum Node.js version required for Gatsby should be v8.x now.

Reply:
Gatsby should support all [LTS releases of Node](https://github.com/nodejs/Release). If there are errors with Node 6 or 7 then that's an issue with Gatsby that should be fixed.